content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -136,7 +136,7 @@ If you are on macOS and using [Homebrew](https://brew.sh/) package manager, you
 1. Test to ensure the version you installed is up-to-date:
 
    ```bash
-   kubectl version --client
+   kubectl version --output=yaml
```

Comment:
This change is not helpful. We want to avoid trying to query a Kubernetes cluster.

Reply:
this query is just to check if kubectl is installed and up-to date in the macos env

Reply:
The original query works even if `kubectl` is not yet configured. The suggested replacement fails (possibly after a timeout) if `kubectl` is not yet configured. We should not make _this_ change.